__tests__/kata6.joinNames.test.js

```diff
@@ -0,0 +1,28 @@
+const { joinNames } = require('../src');
+
+describe('joinNames', () => {
+  it('should return string of names, seperated by commas and an ampersand', () => {
+      expect(joinNames([{
+        name: 'Bart'
+      }, {
+        name: 'Lisa'
+      }, {
+        name: 'Maggie'
+      }])).toBe('Bart, Lisa & Maggie');
+
+      expect(joinNames([{
+        name: 'John'
+      }, {
+        name: 'Paul'
+      }, {
+        name: 'Ringo'
+      }, {name: 'George'
```

Comment:
Bring `name` onto a new line here and ensure the closing curly bracket that follows lines up with the previous closing bracket.